Adyen/Utilities/Observable/AdyenObservable.swift

```diff
@@ -1,13 +1,12 @@
 //
-// Copyright (c) 2021 Adyen N.V.
+// Copyright (c) 2022 Adyen N.V.
 //
 // This file is open source and available under the MIT license. See the LICENSE file for more info.
 //
 
 import Foundation
 
 /// Wraps a value to make it observable.
-/// :nodoc:
 @propertyWrapper
 public final class AdyenObservable<ValueType: Equatable>: EventPublisher {
```

Comment:
Could this be SPI

Reply:
its used in the public interface of `CardSecurityCodeFormatter`, so lets refactor this in a different PR